source/sass-api-reference/index.html.md.erb

```diff
@@ -46,6 +46,13 @@ weight: 65
 <% end %>
 <% end %>
 
+<% if item.context.type == 'variable' %>
+##### Default value
```

Comment:
```suggestion
##### Default
```

Suggest using just 'Default' for this heading, because the example code in each code has more than the value alone.

Reply:
I think I'd be inclined to keep it as is, but I can't really articulate why. Part of me just thinks 'default _what_?' I guess it is showing you what the default value is, it's just that it shows it with a bit more context to help you understand what is being shown.

Thoughts? How strongly do you feel about this one? 😁

Reply:
Not massively! Feel free to keep as 'Default value'.

Reply:
Cool, OK. Let's stick with it for now and we can always revisit.